src/utils/phoenix-utils.js

```diff
@@ -20,7 +20,17 @@ export function connectToReticulum() {
   const socketUrl = `${socketProtocol}//${socketHost}${socketPort ? `:${socketPort}` : ""}/socket`;
   console.log(`Phoenix Socket URL: ${socketUrl}`);
 
-  const socket = new Socket(socketUrl, { params: { session_id: uuid() } });
+  const socketSettings = {
+    params: { session_id: uuid() }
+  };
+
+  if (debug) {
+    socketSettings.logger = (kind, msg, data) => {
+      console.log(`${kind}: ${msg}`, data);
+    };
+  }
```

Comment:
IMO we should use our logging framework and its log levels, although we have historically been half-hearted about it.

Reply:
transitioning to it seems like a dedicated PR -- we have these console logs all over